### PR TITLE
Add a section to introduce pluck's eager loading behavior [ci skip]

### PR DESCRIPTION
**Pull request by st0012**
Personally I found people often don't aware this implicit behavior, so maybe we can raise users attention on this.

### CODE REVIEW

**Comment by st0012** [comment]
ping @sikachu

File: guides/source/active_record_querying.md [hunk 1]
```diff
@@ -1821,6 +1821,21 @@ Client.limit(1).pluck(:name)
 # => ["David"]
 ```
 
+You should also know that using `pluck` will trigger eager loading if the relation object contains include values, even if the eager loading is not necessary for the query. For example:
```
Comment (sikachu) [inline guides/source/active_record_querying.md]: Would you mind prefix this with `NOTE:`?

(Looks like we need support for NOTE block though ... maybe I can add one 🤔)

**Review by sikachu — CHANGES_REQUESTED** [review]

File: guides/source/active_record_querying.md (hunk 1 above)
Comment (st0012) [reply guides/source/active_record_querying.md]: I will add the prefix. Although I totally support the idea of NOTE block as well.

**Comment by st0012** [comment]
@sikachu updated

**Comment by st0012** [comment]
@sikachu would you mind give it a look again?

**Review by sikachu — APPROVED** [review]

**Pull request merged**